diff --git a/bin/yamlsyntax2json b/bin/yamlsyntax2json
index e1de0fbeb54a471e176e425c16a61066a22f3c9d..5bbfa7a069adce0dd6ab110e2aabe7584a7d7ed2 100644
--- a/bin/yamlsyntax2json
+++ b/bin/yamlsyntax2json
@@ -60,9 +60,9 @@ class Main {
         if (orig['patterns'] instanceof Array) r['patterns'] = orig['patterns'].map(r => this.parseRule(r));
         if (typeof orig['contentName'] == 'string') r['contentName'] = orig['contentName'];
         if (typeof orig['include'] == 'string') r['include'] = orig['include'];
-        if (orig['captures'] instanceof Array) r['captures'] = orig['captures'];
-        if (orig['beginCaptures'] instanceof Array) r['beginCaptures'] = orig['beginCaptures'];
-        if (orig['endCaptures'] instanceof Array) r['endCaptures'] = orig['endCaptures'];
+        r['captures'] = orig['captures'];
+        r['beginCaptures'] = orig['beginCaptures'];
+        r['endCaptures'] = orig['endCaptures'];
         return r;
     }
 